lib/Option/CMakeLists.txt

```diff
@@ -6,16 +6,18 @@ add_dependencies(swiftOption
 target_link_libraries(swiftOption PRIVATE
   swiftBasic)
 
-set(features_file_src "${CMAKE_CURRENT_SOURCE_DIR}/features.json")
+set(features_file_swift_src "${CMAKE_CURRENT_SOURCE_DIR}/features.json")
+set(features_file_clang_src "${LLVM_MAIN_SRC_DIR}/../clang/tools/driver/features.json")
```

Comment:
This feels very wrong to me, if there's a better way to do this please let me know 🙇